test/IDE/rdar141440011.swift

```diff
@@ -2,7 +2,7 @@ public protocol MyProto {}
 public struct MyStruct: MyProto {}
 
 // RUN: %empty-directory(%t)
-// RUN: %swift -emit-module -o %t/swift_mod.swiftmodule %s -parse-as-library -experimental-skip-all-function-bodies -experimental-skip-non-exportable-decls -experimental-lazy-typecheck -target %target-triple
+// RUN: %target-swift-frontend -emit-module -o %t/swift_mod.swiftmodule %s -parse-as-library -experimental-skip-all-function-bodies -experimental-skip-non-exportable-decls -experimental-lazy-typecheck -target %target-triple
```

Comment:
Why add in `-target %target-triple` if you aare using `%target-swift-frontend`?

Reply:
Oh good catch, both commands actually